certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json

```diff
@@ -0,0 +1,616 @@
+                    "type": "string",
+                    "description": "Email address of the collaborator",
+                    "title": "Collaborator email"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "Display name of the collaborator",
+                    "title": "Collaborator name"
+                  },
+                  "role": {
+                    "type": "string",
+                    "description": "Role of the collaborator",
+                    "title": "Collaborator role"
+                  },
+                  "canComment": {
+                    "type": "boolean",
+                    "description": "Collaborator permission to comment",
+                    "title": "Collaborator can comment"
+                  },
+                  "canShare": {
+                    "type": "boolean",
+                    "description": "Collaborator permission to share",
+                    "title": "Collaborator can share"
+                  }
+                }
+              },
+              "description": "collaborators"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "operationId": "CCL_LibraryUpdated",
+        "x-ms-trigger": "single",
+        "summary": "When a Creative Cloud Library is updated",
+        "description": "Notify when a Creative Cloud Library is updated.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/clientId_OAuthWeb"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "library_urn": {
+                      "type": "string",
+                      "description": "Choose the Library you want notifications for when it's updated. Select nothing if you want notifications for updates on any Library",
+                      "x-ms-visibility": "important",
+                      "title": "Library",
+                      "x-ms-dynamic-values": {
+                        "operationId": "GetLibrariesList",
+                        "value-path": "library_urn",
+                        "value-title": "name",
+                        "parameters": {
+                          "selector": "all"
+                        }
+                      }
+                    }
+                  },
+                  "description": "data"
+                },
+                "config": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "description": "url",
+                      "title": "",
```

Comment:
@dches Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Added title for this property